partitura/io/__init__.py

```diff
@@ -106,19 +107,19 @@ def load_score(filename: PathLike, force_note_ids="keep") -> Score:
             print(exception)
 
         raise NotSupportedFormatError
-    
-    
+
+
 def lp(filename: PathLike) -> Part:
```

Comment:
should we use a better name for this? We also have the merge parts function in `score`.
Maybe `load_score_as_part`